introduction.md

```diff
@@ -1,3 +1,4 @@
-Hi I am seaseng!!!  AAAArrrrrr (www.yahoo.com)yahoo
-**sodfj**
-Ruby
\ No newline at end of file
+* Git
+* Javascript
+* Ruby
+* SQL
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?